test_cases/wof_localities.json

```diff
@@ -158,12 +158,12 @@
     },
     {
       "id": 8,
-      "status": "fail",
+      "status": "pass",
       "issue": "https://github.com/pelias/acceptance-tests/issues/213",
       "description": "returns the right result second, cause unknown",
       "user": "Stephen",
       "in": {
-        "text": "San Jaun, Puerto Rico",
+        "text": "San Juan, Puerto Rico",
```

Comment:
Woah. Was this a typo we never caught and were confused by? Or did it return the right result somehow.